### ISSUE TITLE
Detection independent tracking

### ISSUE THREAD

**Issue by BornInWater** [abewley/sort#36]
Hi,
So [correct me if i am wrong], this tracking algorithm is heavily dependent on detection in the sense that if the detector fails to detect the object for one frame then the corresponding tracker will be discarded and a new one will be spawned and instantiated the next time the object is detected by the detector. 
Is there a way to ensure retention of tracking for a couple of frames despite non-detection by the detector?

**Comment by abewley** [comment]
You are correct that this tracker (like all tracking by detection methods) are very sensitive to the input detections.
However, to retain the identities for longers you can increase the default max_age parameter which will keep the individual tracks alive for longer to prevent fragmentation. 
See the related issue #22

**Comment by BornInWater** [comment]
<img width="456" alt="screen shot 2018-06-25 at 1 35 40 pm" src="https://user-images.githubusercontent.com/36148827/41837978-b012e596-787c-11e8-9036-936f3c2dcb77.png">

I had tried that before, does not seem to be affecting the output at all.

**Comment by landiaokafeiyan** [comment]
@BornInWater  I changed the max_age to a big number,and It doesn't have much impact on the output, 
but when I changed the min_hits, It has a difference.

**Comment by cygerts** [comment]
@landiaokafeiyan If you want to see difference on max_age just follow this thread:
https://github.com/abewley/sort/issues/55
